src/server/game/AI/ScriptedAI/ScriptedEscortAI.h

```diff
@@ -121,11 +96,11 @@ struct TC_GAME_API npc_escortAI : public ScriptedAI
         uint32 m_uiPlayerCheckTimer;
         uint32 m_uiEscortState;
         float MaxPlayerDistance;
+        uint32 LastWP;
 
-        Quest const* m_pQuestForEscort;                     //generally passed in Start() when regular escort script.
+        WaypointPath _path;
 
-        std::list<Escort_Waypoint> WaypointList;
-        std::list<Escort_Waypoint>::iterator CurrentWP;
+        Quest const* m_pQuestForEscort;                     //generally passed in Start() when regular escort script.
```

Comment:
Hey, i didnt add that - it was already here!
I'll think about that one when im done with other stuff